backend/models/class.model.ts

```diff
@@ -17,6 +17,14 @@ export interface Class extends Document {
   teacher: string;
   /** the ids of the test sessions assigned to the class */
   testSessions: string[];
+  /** the students of the class */
+  students: Student[];
+}
+
+export interface Student {
```

Comment:
could you actually make a Student schema too? and Student should have an `id` field

Reply:
you can reference ResultSchema in testSession.model.ts as an example

Reply:
Changed!